### PR TITLE
[Docs] Avoid port conflict for NGINX-SM and TANZU-SM and update the tanzu-sm.md

### LINKED ISSUE #4117: [Docs] Avoid port conflict with Meshery Adapter for NGINX SM and Tanzu SM
#### Current State
Meshery Adapter for Tanzu SM lists port 10010 as its reserved port. This conflicts with the Meshery Adapter for NGINX SM reserved port.

#### Desired State
Bump Tanzu SM out to 10011. Update this file - https://github.com/meshery/meshery/blob/master/docs/_adapters/tanzu-sm.md

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by SirZeck**
Signed-off-by: adithebest.ar@gmail.com <adithebest.ar@gmail.com>

**Description**

This PR fixes #4117 

**Notes for Reviewers**
Bumped the tanzu-sm port to 10011/tcp and updated the file - https://github.com/meshery/meshery/blob/master/docs/_adapters/tanzu-sm.md

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [Aditya Raj ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bcd9d2b78d14f7ad318ce7fecc52fbca6be95b0d at: https://613994b575b5eea0f988a77f--meshery-docs-preview.netlify.app

File: docs/_adapters/tanzu-sm.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Meshery Adapter for Tanzu Service Mesh
 name: Meshery Adapter for Tanzu Service Mesh
 mesh_name: Tanzu Service Mesh
 version: pre-GA
-port: 10010/tcp
+port: 10011/tcp
```
Comment (alphaX86) [inline docs/_adapters/tanzu-sm.md:7]: Port number is updated. LGTM... pinging other reviewers...

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thanks @SirZeck

**Comment by l5io** [comment]
🚀 Preview for commit d1ba08f4016f763bc035e82b15dd4c6e110fefdc at: https://6139c6284cc966bd10ae91c2--meshery-docs-preview.netlify.app

**Review by sayantan1413 — APPROVED** [review]
LGTM!

**Comment by leecalcote** [comment]
@SirZeck will you consider providing more descriptive titles? 

[area] component: thing that was changed

**Comment by SirZeck** [comment]
> @SirZeck will you consider providing more descriptive titles? 
> 
> [area] component: thing that was changed
> 
> Also, please avoid putting the issue number in the subject line. Including in the issue description is ideal.

Sure, Lee. These changes works? 
Next time I'll surely provide descriptive titles.

**Review by leecalcote — APPROVED** [review]
Thank you. 👍

**Pull request merged**